seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -160,7 +160,7 @@
       "tests": [
         "assert(code.match(/const/g), 'message: Do not replace <code>const</code> keyword.');",
         "assert(code.match(/const\\s+s/g), 'message: <code>s</code> is declared with <code>const</code>.');",
-        "assert(code.match(/const\\s+s\\s*?=\\s*?\\[\\s*?2\\s*?,\\s*?5\\s*?,\\s*?7\\s*?\\]\\s*?;/g), 'message: Do not change the original array declaration.');",
+        "assert(code.match(/const\\s+s\\s*?=\\s*?\\[\\s*?5\\s*?,\\s*?7\\s*?,\\s*?2\\s*?\\]\\s*?;/g), 'message: Do not change the original array declaration.');",
```

Comment:
@Ethan-Arrowood I just noticed this when I was testing something locally so figured I'd mention it before this gets merged: I think this regex can safely be changed to:
```js
code.match(/const\\s+s\\s*=\\s*\\[\\s*5\\s*,\\s*7\\s*,\\s*2\\s*\\]\\s*;?/g)
```
The `?`s are redundant since `*` is already saying the space is optional, and also, this makes the semicolon optional since all we really care about is how the array was declared.